assets/xml/objects/object_oyu.xml

```diff
@@ -1,9 +1,10 @@
 ﻿<Root>
+    <!-- Hot Spring Water found in the Woodfall region and the Empty Ikana Well -->
```

Comment:
Woodfall region? Do you mean the Snowhead region. And are you referring to Beneath the Well? Not sure if you need to put locations in there but maybe its fine

Reply:
Yeah, I meant Snowhead Village I mixed up the names, and I wasn't sure if Beneath the Well was the official name of the Ikana well so I just put that instead. I put the locations just to add more information to whoever plans to use this code later on, people may forget about it being in Beneath the Well so might as well remind them.

Reply:
This object isn't actually used for Beneath the Well at all. This is very specific to Darmani's Gravestone. The hot spring water in beneath the well is actor `z_en_test5.c`

Reply:
Even the hot spring water in the grotto in the snowhead region is `z_en_test5.c` and doesn't use this asset at all

Reply:
Considering what engineer said, maybe this asset should have the prefix `gGoronGraveyardHotSpringWater`? Since it's a very specific asset used for a very specific location.

Reply:
Should I take care to update the other water springs to keep them in line?

Reply:
It's up to you, there isn't really an object file for `z_en_test5.c` since it just uses waterboxes and effects. Plus, this object file is fairly self-containing and would be ready to merge with the updated names. But if there's something you want to document that could use better docs, that's always welcome